src/components/tokenInputPercentage/TokenInputPercentage.tsx

```diff
@@ -0,0 +1,86 @@
+import { BigNumber } from 'bignumber.js';
+import { TokenInputField } from 'components/tokenInputField/TokenInputField';
+import { useState, useMemo, useEffect } from 'react';
+import { Token } from 'services/observables/tokens';
+import { classNameGenerator } from 'utils/pureFunctions';
+
+interface TokenInputPercentageProps {
+  label: string;
+  token?: Token;
+  balance?: string;
+  amount: string;
+  setAmount: Function;
+}
+
+export const TokenInputPercentage = ({
+  token,
+  balance,
+  label,
+  amount,
+  setAmount,
+}: TokenInputPercentageProps) => {
+  const [amountUSD, setAmountUSD] = useState('');
+  const percentages = useMemo(() => [25, 50, 75, 100], []);
```

Comment:
just move percentages above the component as a regular array

Reply:
agreed